example/example.js

```diff
@@ -72,6 +72,7 @@ function postUser() { return; }
  * @apiDescription This function has same errors like POST /user, but errors not defined again, they were included with "apiErrorStructure"
  *
  * @apiParam {String} name Name of the User.
+ * @apiParam {file} avatar upload avatar.
```

Comment:
Use "File" instead of "file" as all other apiParams are capitalized.

Use "Upload avatar." too.

Reply:
ok ,sorry ,i will check again .  thank you so much